config/locales/en.yml

```
@@ -1431,7 +1431,7 @@ en:
       cinstance:
         intentions: "Please describe briefly your intentions of use of this service"
         user_key: "Allowed characters: [A-Z a-z 0-9 - _ .], or Base64 format without forward slash (/), no spaces and up to 256 characters."
-        key: "Allowed characters: [A-Z a-z 0-9 ! \" # $ % & ' ( ) * + , - . : ; < = > ? @ [ \ ] ^ _ ` { | } ~], no spaces and between 5 and 255 characters."
+        key: "Allowed characters: [A-Z a-z 0-9 ! \" # $ % & ' ( ) * + , - . / : ; < = > ? @ [ \ ] ^ _ ` { | } ~], no spaces and between 5 and 255 characters."
```

It seems we already support all special characters ([21 to 7E](https://www.asciitable.com)) and we could take the chance to remove this ugly hint
```suggestion
        key: "No spaces and between 5 and 255 characters."
```

---

Well, I would object that we can say "all"...

"special characters" is a very broad term IMO, and for example those in "The extended ASCII codes" [here](https://www.ascii-code.com/). So, one could try to add `€` or `«`, which would fail.

---

I don't know if there is a better term for those special characters that are not in the "extended" set :upside_down_face:  to make it clear